### DISCUSSION TITLE
New case for IMSAI 8080

### DISCUSSION THREAD

**Discussion by jdourish** [thehighnibble/thehighnibble.github.io#185] [IMSAI8080esp replica]
Hope this is ok to post here, if not please remove.

Have had a new Oak case made for my IMSAI 8080 Front Panel Replica by our local cabinet maker.

![20241231_082006](https://github.com/user-attachments/assets/5bce8f65-8451-46a3-851b-1ea5a3f1c72e)

**Comment by rsiggy** [comment]
Really beautiful work!

**Comment by computer-logix** [comment]
Nice!

I just got my kit the other day (Damn Canada postal strike...) and it arrived with a bent case.   The crucifix parts just don't line up very well so I just might end up making my own case too.

**Reply by jdourish** [reply]
Always found the supplied case to be top heaver and would result in the case falling over. This ended up with the Perspex crucifix slots cracking, hence the need for  new case.

**Reply by computer-logix** [reply]
Mine is misaligned because the box got crushed in a warehouse during the postal strike.   After waiting 4 months to get my hands on one I wasn't sending it back.   The Altar Duino case is much thicker and goes all the way around so the weight distribution is much better and doesn't tip.   They are both amazing kits, but the crucifix is definitely a weak point of the IMSAI kit, even with the reinforcement pieces he's added to the kit.   Some simple L brackets in behind would be better in my opinion.

**Comment by Randall-Routh** [comment]
Beautiful, Nice work.

**Comment by gthelding** [comment]
This is lovely! Very reminiscent of the early days of PCs. I wish I had finish carpentry skills like that. I take it you're using the S-132 for the monitor/keyboard?

**Reply by jdourish** [reply]
I am indeed using the S-132. A nice addition to the IMSAI 8080 will worth the money.

![S-132](https://github.com/user-attachments/assets/aac1b85b-abb3-4469-9b57-0adf384f51c8)
